src/Sulu/Bundle/RouteBundle/Document/Subscriber/RoutableSubscriber.php

```diff
@@ -223,7 +221,7 @@ public function handlePublish(PublishEvent $event): void
         $document->setRoutePath($route->getPath());
         $this->entityManager->persist($route);
 
-        $node->setProperty(
```

Comment:
the `PublishSubscriber` sets the live node to the event. all other subscribers should get the node from the event to process the correct node:
https://github.com/sulu/sulu/blob/59d69249772a78634505aae92152af91e6d86457/src/Sulu/Bundle/PageBundle/Document/Subscriber/PublishSubscriber.php#L79